easybuild/easyconfigs/l/libxc/libxc_3_longdouble.patch

```diff
@@ -0,0 +1,29 @@
+# Patch to remove long double suffix from some constants so the library can compile in POWER architectures.
+# This does not change the results obtained with the library, as long doubles are not used anywhere else.
```

Comment:
@micaeljtoliveira Do you have a link to the upstream fix you're backporting here? Same for the other patch (you'll need to fix the checksums for the patches too, `eb --inject-checksums` is your friend here)

Reply:
The [upstream fix](https://gitlab.com/libxc/libxc/commit/10889c4c32db6c735715782f66895a24b626bfee) to the 3.x.x. and 4.0.x releases is not the same as the one I implemented here, as that was a simple workaround that did not correct the actual problem. This problem with the long doubles reappeared later and was then fixed correctly [here](https://gitlab.com/libxc/libxc/commit/de20fdea12819f3ff90aad71a0e67475c29cce51).

Putting these links in the patch without this longer explanation will probably be more confusing then anything else.

Reply:
@micaeljtoliveira I think referring to https://gitlab.com/libxc/libxc/commit/de20fdea12819f3ff90aad71a0e67475c29cce51 is good enough, it confirms that this same change was made upstream at some point, and provides a handle to figure out more...